### ISSUE TITLE
Templates are stuck on version 11.1.0

### ISSUE THREAD

**Issue by StefH** [ChilliCream/hotchocolate#3896]
**Describe the bug**
Friends is null when getting character ?

**To Reproduce**
1]
- `dotnet new --install "HotChocolate.Templates.StarWars"`
- `dotnet new starwars`

2]
Start server

3] Execute this GraphQL
``` gql
query {
  character(ids: [1000, 1001])  {
    id
    name
    friends {
      nodes {
        name
      }
    }
  }
}
```

This results in:
``` json
{
  "errors": [
    {
      "message": "Cannot handle the specified data source."
    },
    {
      "message": "Cannot handle the specified data source."
    }
  ],
  "data": {
    "character": [
      {
        "id": 1000,
        "name": "Luke Skywalker",
        "friends": null
      },
      {
        "id": 1001,
        "name": "Darth Vader",
        "friends": null
      }
    ]
  }
}
```

**Expected behavior**
The friend's names are returned as a list

**Desktop (please complete the following information):**
Tool version :
`11.3.0`

NuGet versions
``` xml
<ItemGroup>
    <PackageReference Include="HotChocolate.AspNetCore" Version="11.1.0" />
    <PackageReference Include="HotChocolate.AspNetCore.Authorization" Version="11.1.0" />
    <PackageReference Include="HotChocolate.Data" Version="11.1.0" />
  </ItemGroup>
```

**Comment by StefH** [comment]
Upgrading the NuGets to:
``` xml
<ItemGroup>
    <PackageReference Include="HotChocolate.AspNetCore" Version="11.3.1" />
    <PackageReference Include="HotChocolate.AspNetCore.Authorization" Version="11.3.1" />
    <PackageReference Include="HotChocolate.Data" Version="11.3.1" />
  </ItemGroup>
```

Solved the issue.

**Comment by tobias-tengler** [comment]
@michaelstaib I thought we fixed the templates already, by doing this:
https://github.com/ChilliCream/hotchocolate/blob/b73f0abfb5efee94065dbf0f116826dc4423c51c/.build/Build.Publish.cs#L33
But the published version still contains the version 11.1.0.
We should probably fix this asap.

**Comment by michaelstaib** [comment]
I have put it into the v12 release backlog.
